rc/lsp.kak

```diff
@@ -973,16 +973,20 @@ define-command -hidden lsp-show-hover -params 3 -docstring %{
 } %{ evaluate-commands %sh{
     lsp_info=$2
     lsp_diagnostics=$3
+
+    # To make sure we always show diagnostics, restrict only the info portion based
+    # on the configured maximum line count
+    if [ $kak_opt_lsp_hover_max_lines -gt 0 ]; then
+        diagnostics_count=$(printf "%s", "$lsp_diagnostics" | wc -l)
+        lsp_info=$(printf %s "$lsp_info" | head -n $(($kak_opt_lsp_hover_max_lines - 2 - $diagnostics_count)))
```

Comment:
We should probably find a cleaner fix here eventually.
This currently discards 2 lines even if there are no diagnostics (I guess that can be fixed easily).
More importantly, this also discards lines even if the user doesn't render diagnostics in `kak_opt_lsp_show_hover_format`.
Though I'm not sure why anyone would do that, so I guess it's fine.